packages/core/src/context/hotkeys/hotkeys-provider.md

```diff
@@ -103,15 +104,20 @@ function Plugin() {
     );
 }
 
-function PluginSlot(props) {
+function PluginSlot({ children }: { children: React.ReactNode }) {
```

Comment:
N.B. I haven't actually tested/executed this code, so I'm not 100% sure if it is right for this given example for plugins. At the very least, the code does compile TS.